requirements.txt

```diff
@@ -1,7 +1,7 @@
 Django==1.11.29
 djangorestframework==3.9.4
 jsonschema==2.5.0
-uwsgi==2.0.18
+uwsgi==2.0.19.1
```

Comment:
Maybe this kind of change should be added to Changelog?

For example we have this in the Changelog, related with other libraries:

```
UPDATE: Update Django from 1.11.28 to 1.11.29 due to security vulnerability
```